packages/assets/src/Assets.php

```diff
@@ -1,10 +1,22 @@
 <?php
+/**
+ * Jetpack Assets package.
+ *
+ * @package  automattic/jetpack-assets
```

Comment:
I'm not sure if we're using this nomenclature in packages or still going with `Jetpack`.

Reply:
I think we've been using this in other packages as well. 👍